index.html

```diff
@@ -434,11 +434,117 @@ <h2 id="protocol-registry">
       the future.
     </p>
     <h3>
-      Inclusion criteria
+      General inclusion criteria
     </h3>
+    <aside class="note">
+      The below criteria are a work in progress and are likely to change as
+      this document evolves.
+    </aside>
     <p>
-      To be included in the registry...
+      To be included in the registry, the [=digital credential/exchange
+      protocol=]:
     </p>
+    <ol>
+      <li>MUST be standardized at a <a href=
+      "https://www.w3.org/liaisons/">consortium the W3C liaises with</a>
+      </li>
+      <li>MUST have a stable URL that points to a publicly available
```

Comment:
```suggestion
      <li>MUST have a stable URL that points to a freely and publicly available
```

Reply:
related to https://github.com/WICG/digital-credentials/pull/157#discussion_r2036167357